### PR TITLE
Enforce validateIndentation

### PR DESCRIPTION
**Pull request by samarsault**

### CODE REVIEW

**Comment by namangoel1** [comment]
@samarjeet27 Looks great, I'll merge if Travis says it's okay :)

**Comment by samarsault** [comment]
wohoo !!! travis passed :D

**Comment by namangoel1** [comment]
@samarjeet27 Awesome work here :D

File: javascripts/main.js [hunk 1]
```diff
@@ -1,199 +1,199 @@
...
+        menu().hide();
     }
+  });
 
-    function set_map_wrap() {
-        $('.map-wrap').css('display', 'inline');
-        console.log('clicked');
+  function validateEmail(m) {
+    if (typeof(m) == 'undefined') {
+      m = $('.email').val();
```
Comment (roonyh) [inline javascripts/main.js:182]: whats going on here. this doesn't seems to be related to indentation?

Comment (namangoel1) [inline javascripts/main.js:182]: I think he's changed the order some, but he's removed 170 lines and added 170 lines.

Comment (roonyh) [inline javascripts/main.js:182]: hmm. no problem. but its best if this is done in a different commit. this commit is about fixing indentation.
